csrankings-r.csv

```diff
@@ -301,6 +301,7 @@ Ran Ben-Basat,University College London,https://www.bbasat.com,6G61qDwAAAAJ
 Ran Canetti,Boston University,http://www.cs.bu.edu/fac/canetti,NOSCHOLARPAGE
 Ran Duan,Tsinghua University,http://iiis.tsinghua.edu.cn/zh/duanr,NOSCHOLARPAGE
 Ran El-Yaniv,Technion,http://www.cs.technion.ac.il/~rani,D9eVSd8AAAAJ
+Ran Gilad-Bachrach,Tel Aviv University,https://english.tau.ac.il/profile/rgb,nnLiId8AAAAJ

```

Comment:
Biomedical engineering, not CS.

Reply:
The added faculty from bio engineering (Ran Gilad Bachrach) is allowed to supervise graduate students in computer science

Reply:
See above.